### DISCUSSION TITLE
Why no MultiMetricExporter?

### DISCUSSION THREAD

**Discussion by huntc** [open-telemetry/opentelemetry-java#3301] [Q&A]
I see there's a MultiSpanExporter. Is there any particular reason for there not to be a MultiMetricExporter? Thanks.

**Comment by jkwatson** [comment]
No one has asked for it? What is your use case for one?

**Reply by huntc** [reply]
Thanks for replying. I had wanted to log metrics at the same time as exporting them elsewhere. It is very easy for me to do this by throwing together my own multi-exporter, but I was more curious as to the asymmetry of why there would be one for spans but none for metrics. No biggy though.

**Reply by jkwatson** [reply]
Mostly that metrics are very incomplete at the moment,I think. 😀
